packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh

```diff
@@ -91,7 +91,7 @@
           !ifdef INSTALL_MODE_PER_ALL_USERS
             nsExec::Exec `taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid"`
           !else
-            nsExec::Exec `%SYSTEMROOT%\System32\cmd.exe /c taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
+            nsExec::Exec `"$SYSDIR\cmd.exe" /c taskkill /f /im "${APP_EXECUTABLE_FILENAME}" /fi "PID ne $pid" /fi "USERNAME eq %USERNAME%"`
```

Comment:
What is `$SYSDIR`? I don't see it when I get env vars in powershell or command prompt

Reply:
$SYSDIR is predefined in NSIS scripts. It's an NSIS var. 
https://nsis.sourceforge.io/Docs/Chapter4.html
<img width="791" alt="image" src="https://github.com/electron-userland/electron-builder/assets/5462444/56b6a506-0284-4721-8dd1-af2064a1b0e7">

Reply:
Nice! Thank you for clarifying